db/seeds.rb

```diff
@@ -8,14 +8,106 @@
 
 #USERS SEEDS--------------------------------------------------------------------
 
-names = ['daniels', 'dupont', 'durand', 'cuvette', 'sentbon', 'toulemonde', 'abidbol']
-surnames = ['juliette', 'vincent', 'manuel', 'romain', 'sophie', 'claire',
-            'maxime', 'pierre', 'jean', 'paul', 'jack', 'claudie']
-operator = ['gmail.com', 'hotmail.fr', 'yahoo.com', 'lewagon.org']
+def new_user
+  names = ['daniels', 'dupont', 'durand', 'cuvette', 'sentbon', 'toulemonde', 'abidbol']
+  surnames = ['juliette', 'vincent', 'manuel', 'romain', 'sophie', 'claire',
+              'maxime', 'pierre', 'jean', 'paul', 'jack', 'claudie']
+  operators = ['gmail.com', 'hotmail.fr', 'yahoo.com', 'lewagon.org']
 
-User.destroy_all
 
-10.times do
-  User.create(email: "#{ surnames.sample }-#{ names.sample }@#{ operator.sample }",
+  User.create(email: "#{ surnames.sample }-#{ names.sample }@#{ operators.sample }",
     password: 'xxx')
 end
+
+
+#WCS SEEDS----------------------------------------------------------------------
+
+
+wcs_attributes = [
+  {
+  name: 'Super Wc',
+  description: 'Wc propre et bien aéré',
+  address: '121 Rue Saint-André, 59800 Lille, France',
+  availability: true
+  },
+  {
+  name: 'Idéal gens pressés',
+  description: 'se trouve dans la cour, donc accès rapide',
+  address: '59-61 Rue du Metz, 59800 Lille, France',
+  availability: true
+  },
+  {
+  name: 'Belle Cuvette',
+  description: 'Toilettes refaits à neuf en début d\'année',
+  address: '32-36 Rue Saint-Sébastien, 59800 Lille, France',
+  availability: true
+  },
+  {
+  name: 'P\'tit coin d\'tranquilité',
+  description: 'WC au calme pour faire son affaire tranquille',
+  address: '2-4 Rue du Béguinage, 59800 Lille, France',
+  availability: true
+  },
+  {
+  name: 'Bon toilette',
+  description: 'Toilet de bon standing, pas de fuite à signaler',
+  address: '1 Rue du Metz, 59800 Lille, France',
+  availability: true
+  },
+  {
+  name: 'WC propres',
+  description: 'Ménage fait après chaque passage',
+  address: '15 Rue de la Halle, 59800 Lille, France',
+  availability: true
```

Comment:
Maybe set some of the wc 2-3 with an availability at false... could be useful for some tests.